patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -314,6 +439,18 @@ function toggleMenu() {
 function closeMenu() {
   commonCloseMenuTasks();
   menuButton.setAttribute("aria-pressed", "false");
+
+  // Set focusn on the menu button.
+  setTimeout(function timeoutFunction() {
+    document.querySelector(".js-header-menu-button").focus();
+  }, 100);
+
+  if ((width > 840) && document.querySelector(".js-utility-nav--wide .ma__utility-nav__item .goog-te-menu-value").hasAttribute("tabindex")) {
```

Comment:
@ygannett Currently getting an error o this line with the value of the querySelector reporting as null. This prevents me from closing the hamburger menu.

Reply:
@tom-fleming Thank you for pointing that out.  That is JS generated item by Google.  It seems the item is not ready (rendered) sometimes when this script runs.  So, I switched the item with the hard coded one in the template to prevent the error.